notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>I think we should comment this. We have a loss function MAE and we are optimizing with respect to precision at k. So basically, our model should have a good precision at k. If we are looking to optimize a different metric, such us recall, we can just add here recall and the results will be different. </p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I think this code (along with the previous code setting a metric) gives confusion. This is just showcasing an evaluation hook, which reports the given metric values during the training and records them into TensorBoard.

To make the model optimized into a custom metric other than the default, `loss`, should define own `Estimator`. Would not be too difficult, but rather do that on a deep-dive notebook. 

I think would be better to clean up the codes to be:
- remove the previous MAE part (since we anyway do the separate evaluation at the end)
- change the hook name to `precision_log_hook` so that show its purpose clearly, and
- add explanation what the hook does.

Reply:
Added more contexts to the markdown cell